code/modules/tutorials/flock_tutorial.dm

```diff
@@ -0,0 +1,421 @@
+	structure_type = /obj/flock_structure/interceptor
+
+	SetUp()
+		..()
+		src.ftutorial.fowner.flock.achieve(FLOCK_ACHIEVEMENT_BULLETS_HIT)
+		src.ftutorial.portal_in(get_turf(src.ftutorial.center), /mob/living/carbon/human/normal/chef/shoot_gun_person/)
+		src.location = locate(src.ftutorial.center.x - 1, src.ftutorial.center.y - 3, src.ftutorial.center.z)
+		location.UpdateOverlays(marker, "marker")
+
+	PerformAction(action, context)
+		. = ..()
+		if (action == FLOCK_ACTION_TEALPRINT_COMPLETE)
+			var/obj/flock_structure/interceptor/struct = context
+			OVERRIDE_COOLDOWN(struct, "bolt_gen_time", 0 SECONDS)
+			struct.power_projectile_checkers(TRUE)
+
+/datum/tutorialStep/flock/turret_demo
+	name = "Intercept"
+	instructions = "Watch the Interceptor destroy a bullet."
+	SetUp()
+		..()
+		var/turf/T = locate(src.ftutorial.center.x, src.ftutorial.center.y - 6, src.ftutorial.center.z)
+		var/obj/deployable_turret/riot/turret = new(T)
+		turret.set_angle(0)
+		turret.set_projectile()
+		sleep(1 SECOND)
+		muzzle_flash_any(turret, 0, "muzzle_flash")
+		shoot_projectile_ST_pixel_spread(turret, turret.current_projectile, src.ftutorial.center, 0, 0 , turret.spread)
+		SPAWN(10 SECONDS)
+			src.ftutorial.Advance()
+
+/datum/tutorialStep/flock/relay
+	name = "The Relay"
+	New()
+		src.instructions = "This is the Relay, your ultimate goal. Unlocked at [FLOCK_RELAY_COMPUTE_COST] compute, when complete it will allow you to transmit the Signal and cast the Flock out towards our next target."
+		..()
+
+	SetUp()
+		..()
+		var/turf/T = get_turf(src.ftutorial.center)
+		for (var/obj/flock_structure/cage/cage in range(3, T))
+			qdel(cage.occupant)
+			qdel(cage)
+		var/obj/flock_structure/ghost/tealprint = new(T, src.ftutorial.fowner.flock, /obj/flock_structure/relay)
+		tealprint.fake = TRUE
+
+/datum/tutorialStep/flock/showcase
+	name = "Structure showcase"
+	instructions = "Here are all the flock structures you can create, along with a shadow of the relay, your ultimate goal. Click the exit tutorial button in the bottom right corner to exit the tutorial."
+	SetUp()
+		..()
+		src.ftutorial.fowner.abilityHolder.addAbility(/datum/targetable/flockmindAbility/tutorial_exit)
+		var/datum/mapPrefab/allocated/prefab = get_singleton(/datum/mapPrefab/allocated/flock_showcase)
+		var/datum/allocated_region/region = prefab.load()
+		for (var/turf/T in REGION_TILES(region))
+			var/obj/spawner/flock_structure/structure_spawner = locate() in T
+			structure_spawner?.spawn_structure(ftutorial.fowner.flock)
+			var/mob/living/carbon/human/bad_immortal/fake_tdummy = locate() in T
+			if (fake_tdummy)
+				src.ftutorial.fowner.flock.updateEnemy(fake_tdummy)
+		for(var/turf/T in landmarks[LANDMARK_TUTORIAL_START])
+			if(region.turf_in_region(T))
+				src.ftutorial.fowner.set_loc(T)
+				break
+		src.ftutorial.ui_interact(src.ftutorial.fowner)
+
+/mob/living/intangible/flock/flockmind/verb/help_my_tutorial_is_being_a_massive_shit()
```

Comment:
Don't think this will work if the tutorial breaks, if that's possible, while you are in the controlling a drone stages of the tutorial?